### ISSUE TITLE
torchtext 0.16.0 wheels are missing for aarch64 linux platform

### ISSUE THREAD

**Issue by snadampal** [pytorch/text#2208]
## 🐛 Bug

**Describe the bug** A clear and concise description of what the bug is.
torchtext 0.16.0 wheels are missing for aarch64 linux platform. This is breaking the torchbench benchmarks on aarch64 linux, because (1) torchbench requires torchtext and (2) torchtext 0.15.2 is not compatible with torch 2.1.0

**To Reproduce** Steps to reproduce the behavior:

```
# Install PyTorch and extensions
python3 -m pip install torch
python3 -m pip install torchvision torchaudio torchtext

# Clone PyTorch benchmark repo
git clone https://github.com/pytorch/benchmark.git

# Setup Resnet50 benchmark
cd benchmark
python3 install.py resnet50
```

**Expected behavior** A clear and concise description of what you expected to happen.
torchtext 0.16.0 wheels are published for aarch64 linux platform

**Screenshots** If applicable, add screenshots to help explain your problem.

**Environment**
AWS Graviton3 with Ubuntu 22.04

Please copy and paste the output from our
[environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py) (or
fill out the checklist below manually).

You can get the script and run it with:

```
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
python -c "import torchtext; print(\"torchtext version is \", torchtext.__version__)"
```

- PyTorch Version (e.g., 1.0): 2.1.0
- OS (e.g., Linux): Ubuntu 22.04
- How you installed PyTorch (`conda`, `pip`, source): pip
- Build command you used (if compiling from source):
- Python version: 3.10
- CUDA/cuDNN version:
- GPU models and configuration:
- Any other relevant information:

**Additional context** Add any other context about the problem here.

**Comment by malfet** [comment]
Yes, this results from a fact that torchtext has never been added to the automated pipeline that were build for torchvision and torchaudio. On the other hand, `torchtext` should not be a mandatory dependency for the latest torchbench, should it?

**Comment by snadampal** [comment]
thanks, @malfet. Sorry, my scripts are using the torchbench from a commit, but looks like the dependency is removed in the latest torchbench. 
If torchtext dependency is deprecated for every project, then we are good. Otherwise, there is a need for the updated wheels that are compatible with torch 2.1.
